packages/plugin-node-uncaught-exception/test/uncaught-exception.test.js

```diff
@@ -7,30 +7,31 @@ const plugin = require('../')
 describe('plugin: node uncaught exception handler', () => {
   it('should listen to the process#uncaughtException event', () => {
     const before = process.listeners('uncaughtException').length
-    const c = new Client({ apiKey: 'api_key' })
-    c.use(plugin)
+    const c = new Client({ apiKey: 'api_key', plugins: [plugin] })
     const after = process.listeners('uncaughtException').length
     expect(after - before).toBe(1)
+    expect(c).toBe(c)
```

Comment:
What is the purpose of these assertions?

Reply:
Appeasing the linter :(

instantiating a client and then not referring to it leads to an unused variable error. If I remove the assignment to `client`, it errors with the "don't use new for side effects" rule.

I didn't want to ignore the line where it was instantiated, so this seemed like the least bad option. I'm sure there's a better way.

Reply:
Fair enough. We can revisit this as part of the TS conversion.